lib/startback/tracing/operation_tracer.rb

```diff
@@ -0,0 +1,14 @@
+module Startback
+  module Tracing
+    class OperationTracer
+      def call(runner, op)
+        op.context.tracer.fork({
+          :operation => op.class.name,
+          :context => op.context.to_h
```

Comment:
The exact data here will be part of the public interface, and should be chosen wisely to avoid too many breaking changes later.

For instance no operation data is tracked currently. Should we add it, like the Audit::Trailer does? Or avoid that first and add it later if we see that it's definitely missing?